sdk/metricsadvisor/ai-metrics-advisor/samples/typescript/src/alertingConfig.ts

```diff
@@ -84,18 +86,25 @@ async function updateAlertConfig(
   detectionConfigId: string,
   hookIds: string[]
 ) {
-  const metricAlertingConfig: MetricAlertConfiguration = {
-    detectionConfigurationId: detectionConfigId,
-    alertScope: {
-      scopeType: "All"
-    }
-  };
   const patch: Omit<AnomalyAlertConfiguration, "id"> = {
     name: "new Name",
     //description: "new description",
     hookIds,
     crossMetricsOperator: "OR",
-    metricAlertConfigurations: [metricAlertingConfig, metricAlertingConfig]
+    metricAlertConfigurations: [
+      {
+        detectionConfigurationId: detectionConfigId,
+        alertScope: {
+          scopeType: "All"
+        }
+      },
+      {
```

Comment:
could we use some different values for the second metric alerting config?

Reply:
done